src/components/Fund/Fund.js

```diff
@@ -212,7 +212,11 @@ class Fund extends Component {
     };
 
     return (
-      <div style={{ width: '100%' }} className={css.panepadding}>
+      <div
+        style={{ width: '100%' }}
+        className={css.panepadding}
```

Comment:
Do you need to use inline styles and CSS?

Reply:
dunno, ask Arvind :)